plan/physical_plan_builder.go

```diff
@@ -645,7 +669,8 @@ func (p *Aggregation) handleStreamAgg(prop *requiredProperty) (*physicalPlanInfo
 	return info, nil
 }
 
-func (p *Aggregation) handleFinalAgg(x physicalDistSQLPlan, childInfo *physicalPlanInfo) *physicalPlanInfo {
+// convert2PhysicalPlanFinalHash converts logical aggregation to final hash aggregation *physicalPlanInfo.
```

Comment:
logical aggregation to final hash aggregation *physicalPlanInfo ->
the logical aggregation to the final hash aggregation *physicalPlanInfo